test/controllers/IncomeControllerSpec.scala

```diff
@@ -555,8 +617,8 @@ class IncomeControllerSpec extends PlaySpec
   val employmentAmount = EmploymentAmount("employment","(Current employer)",1,1111,1111,
     None,None,Some(new LocalDate(2000, 5, 20)), None, true, false)
 
-  private def createSUT = new SUT
-  private class SUT extends IncomeController {
+  private def createTestController = new TestController
+  private class TestController extends IncomeController {
```

Comment:
perhaps the new class name should be TestIncomeController rather than just TestController to give a little more meaning to what it is. This was one of the problems with SUT. You didn't know what it was on first glance, you would have to scroll to the bottom of the page to find out.